src/components/tl/mlx5/mcast/tl_mlx5_mcast.h

```diff
@@ -91,6 +99,10 @@ typedef struct mcast_coll_comm_init_spec {
     int                               wsize;
     int                               max_eager;
     void                             *oob;
+    int                               one_sided_reliability_enabled;
```

Comment:
How are these options supposed to be set? Are they configurable?

Reply:
Yes, there will env vars to that set them, they will be added in the future PRs.